### ISSUE TITLE
🐛 Rome hangs on config push

### ISSUE THREAD

**Issue by JustBeYou** [rome/tools#1588]
When running `rome config push` on a fresh project, Rome hangs. I reproduced the bug on two machines running Fedora and Manjaro (on bash and zsh).

## Environment

```
$ rome rage --summary

Environment Variables
  {COLORTERM: "truecolor", LANG: "en_US.UTF-8", TERM: "xterm-256color"}

User Config
  unset

Rome Version
  10.0.7-beta.18

Node Version
  15.11.0

Platform
  linux x64 5.12.11-200.fc33.x86_64

Terminal Features
  {
    background: "unknown"
    colorDepth: 24
    cursor: true
    hyperlinks: true
    isTTY: true
    progress: true
    unicode: true
    columns: OneIndexedNumber<112>
  }

Client Flags
  {
    clientName: "cli"
    silent: false
    cwd: AbsoluteFilePath</tmp/rome>
    realCwd: AbsoluteFilePath</tmp/rome>
  }

Server Status
  {
    projects: [{id: 0}, {id: 1}]
    workers: [
      {
        astCacheSize: 0
        heapTotal: 47_525_888
        ownedBytes: 0n
        ownedFileCount: 0
        pid: 43_197
        uptime: 0.471970752
      }
    ]
    server: {heapTotal: 47_525_888, pid: 43_197, uptime: 0.472116339}
  }
```

## Steps To Reproduce

Run the following commands to reproduce it:
```bash
cd /tmp
git clone --depth 1 https://github.com/romefrontend/rome
mkdir test_prj
cd test_prj
npm init
../rome/rome init # press enter to any question, default config
../rome/rome config push lint.globals beforeEach
# rome hangs here
```

## Expected Results

Rome should exit cleanly instead of hanging.

**Comment by ematipico** [comment]
It's probably a OS thing... I can't reproduce it on macOS. I will check later one when I have a linux based OS.

**Comment by JustBeYou** [comment]
I reproduced it on `WSL 2 Ubuntu 20.04` with `node stable` too.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
